nestkernel/conn_builder.h

```diff
@@ -222,8 +222,8 @@ class ConnBuilder
   //! dictionaries to pass to connect function, one per thread
   std::vector< DictionaryDatum > param_dicts_;
 
-  //! empty dictionary to pass to connect function
-  const static DictionaryDatum dummy_param_;
+  //! empty dictionary to pass to connect function, one per thread
```

Comment:
@stinebuu Could you add a note to the comment to explain why we need one dummy dict per thread? Just so we remember ten years from now.

Reply:
@heplesser done!